docs/user-docs/latest/content-types-builder/creating-new-content-type.md

```diff
@@ -0,0 +1,50 @@
+# Creating content-types
+
+::: warning The Content-Types Builder is only accessible when your Strapi application is in a development environment.
+<br>
+:::
+
+The Content-Types Builder allows to create new content-types: single and collection types. Although they are not proper content-types as they cannot exist independently, components can also be created through the Content-Types Builder, in the same way as collection and single types.
+
+## Creating a new content-type
+
+![Content-type creation](../assets/content-types-builder/content-type-creation.png)
+
+Content types are created from the Collection types and Single types categories of the Content-Types Builder.
+
+To create a new content-type:
+
+1. Choose whether you want to create a collection type or a single type.
+2. Click on the **Create a new collection/single type** button in the category of the content-type you want to create.
+3. In the content-type creation window, write the name of the new content-type in the *Display name* textbox. When creating a collection type, opt for a singular name, as collection type names are automatically pluralised when displayed in the Content Manager.
```

Comment:
Hm I'm not sure it's very good practice to mention lots of GitHub issues, but especially for topics like these where there's apparently no workaround. I'm afraid it'll frustrate users more than help them if we linked this issue here 🤔  I guess since we should fix this issue in the next few months, I won't add anything in the documentation for that topic.